service/service.py

```diff
@@ -273,6 +273,31 @@ def list_items_in_wishlist(wishlist_id):
     return make_response(jsonify(results), status.HTTP_200_OK)
 
 
+######################################################################
+# DELETE ITEM FROM A WISHLIST
+######################################################################
+@app.route("/wishlists/<int:wishlist_id>/items/<int:item_id>", methods=["DELETE"])
+def delete_item_from_wishlist(wishlist_id, item_id):
+    """
+    Deletes an item from a Wishlist
+    This endpoint will return an Item based on its id
+    """
+    app.logger.info("Request to delete an item from a wishlist")
+    wishlist = Wishlist.find_or_404(wishlist_id)
+    get_item = None
+    for item in wishlist.items:
+        if item.id == item_id:
+            get_item = item
+            break
+
+    if get_item is None:
+        raise NotFound("Item with id '{}' was not found.".format(item_id))
+
+    get_item.delete()
+    app.logger.info("Item with ID [%s] was deleted.", wishlist_id)
+    message = wishlist.serialize()
+    return make_response(jsonify(message), status.HTTP_200_OK)
```

Comment:
Delete should return 201

Reply:
201 is Created...I suppose you are referring to 204 No Content? I also have the same thoughts and try to replace ``` status.204_NO_CONTENT ``` here and check the return message with Postman. I was expecting that I could see the wishlist after deleting the item but in fact, I saw nothing. I google for 204 and find out that with this status code, the message body may be set to be empty by default. I think as a user, I would be more willing to see that I successfully delete the item and it is indeed not on the wishlist. So maybe 200 is better. May I ask for your second thought?

Reply:
Yes, I meant 201. A delete API should always return a 201, irrespective of deleting or not.

Reply:
Sure, I can get your point. Maybe we can wait for Jesse to respond and see his opinion. After all, he is the author of this pull request. I think he is currently having classes.

Reply:
I can see what you are saying, but then the same logic applies for any delete, I would want to see my remaining items, and hence, it would never return 204.

Reply:
Resolved with modifying status code from 200 to 204.